### DISCUSSION TITLE
Build for aarch64?

### DISCUSSION THREAD

**Discussion by TheQu1etOwl** [zigtools/zls#499] [Q&A] [answered]
Where is the build for linux-aarch64?

**Comment by truemedian** [comment, marked as answer]
The releases of zls are ancient at this point because zls now tracks the latest master branch of zig. Therefore, to get a build of zls you should build it yourself.

**Reply by TheQu1etOwl** [reply]
[zls-aarch64-linux.zip](https://github.com/zigtools/zls/files/8845663/zls-aarch64-linux.zip)
My build.
